heroku/cli
# refactor: migrate apps:join to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001avfAoYAI/view)

## Testing
Testing this one requires some set up. 
- Someone will have to add you to a Heroku org as a member, viewer, or collaborator (not an admin) and then add you to an app.
- Copy the name of the app
- You will then have to use Dashboard to remove yourself from the app (not the team, just the app)
- Check out this branch and run `yarn` and `yarn build`
- Run `./bin/run apps:join -a APP-NAME`
